visualizer/index.html

```diff
@@ -161,6 +171,105 @@
       color: white;
     }
 
+    .semanticsEditor.hidden .header,
+    .semanticsEditor.hidden .body,
+    .semanticsEditor.hidden .save {
+      display: none;
```

Comment:
For the editor that has classname 'hidden' I want to show the result only. I could see that 'hidden' is a confusing classname here, maybe change to 'resultOnly'?